### DISCUSSION TITLE
After "npm run preview" I got a blank page.

### DISCUSSION THREAD

**Discussion by RhythmPony** [drawdb-io/drawdb#92] [Q&A]
When I deploy dist to server,the page reads "looking for something you couldn't find?"

**Comment by 1ilit** [comment]
If it was built successfully and is loading the NotFound page I assume it's somethings with there router. Maybe it's because there is no base name so the router is failing to recognize the route names with `/`. Can you try adding base to `vite.config.js` like this

```js
export default defineConfig({
  base: './',
  plugins: [react()],
})
```

**Reply by RhythmPony** [reply]
Thanks

**Reply by 1ilit** [reply]
Let me know if it still doesn't work

**Reply by RhythmPony** [reply]
I modified the config as you said,but it still doesn't work.

**Reply by newpgit** [reply]
确实是可以的，我刚刚试了

Yes, I can. I just tried

**Comment by Shahab094** [comment]
I am adding some functional plugins on a[ Science study](https://guideplus.in/8th-science-guide/) simple site that I have as a the project but still facing the same like you. Finding more solutions but no one is working for me.

**Reply by 1ilit** [reply]
Did you try setting the base name in the browser  router to "/" or whatever the base name should be in your case

**Comment by abdulah-hash** [comment]
You can check the further information [here.](https://investguide.fr/)
